### PR TITLE
feat: add get_paths_from_man_locations

### PR DESCRIPTION
**Pull request by pickfire**
Rust implementation does not sort man paths.

### CODE REVIEW

**Comment by pickfire** [comment]
@scooter-dangle Any news on this?

**Comment by scooter-dangle** [comment]
> @scooter-dangle Any news on this?

Hey, @pickfire! Did you see my note on sorting? (https://github.com/rust-dc/fish-manpage-completions/pull/57#discussion_r325055276)

**Comment by pickfire** [comment]
> Hey, @pickfire! Did you see my note on sorting? (#57 (comment))

@scooter-dangle Yes and I replied.

File: src/main.rs [hunk 1]
```diff
@@ -1489,49 +1490,47 @@ impl App {
...
+        eprintln!("Unable to get the manpath, falling back to /usr/share/man:/usr/local/share/man. Please set $MANPATH if that is not correct.");
+        parent_paths.push(PathBuf::from("/usr/share/man"));
+        parent_paths.push(PathBuf::from("/usr/local/share/man"));
+    }
+    let mut paths = Vec::new();
+    for parent_path in parent_paths {
+        for section in &["man1", "man6", "man8"] {
+            let section_path = parent_path.join(section);
+            if let Ok(dir) = fs::read_dir(&section_path) {
+                for entry in dir {
```
[the remark this answers is not in the document]
Comment (scooter-dangle) [reply src/main.rs]: Per comment in main thread, we're leaving off for now since `dir` is an iterator rather than a sortable collection.

**Pull request merged**